app/extras/emojifier.rb

```diff
@@ -2,6 +2,8 @@ class Emojifier
   extend JSRuntime
 
   def self.emojify!(text)
-    eval "emojione.shortnameToImage(\"?\")", text
+    root_url = Rails.application.routes.url_helpers.root_url(ActionMailer::Base.default_url_options)
```

Comment:
This is pretty horrid. I'd be open to hearing thoughts on a better way to do this.